flexinput/src/main/java/com/lytefast/flexinput/fragment/PhotosFragment.kt

```diff
@@ -91,7 +98,7 @@ open class PhotosFragment : PermissionsFragment() {
   private fun requestPermissions(photoAdapter: PhotoCursorAdapter) {
     requestPermissions(object : PermissionsFragment.PermissionsResultCallback {
       override fun granted() {
-        recyclerView!!.layoutManager = GridLayoutManager(context, 3)
+        recyclerView!!.layoutManager = GridLayoutManager(context, GRID_LAYOUT_SPAN_COUNT)
```

Comment:
doesn't this only need to be set once now that I look at it?

Reply:
no, should be at least twice.